### DISCUSSION TITLE
Sort through different quotes that conflict

### DISCUSSION THREAD

**Discussion by pederdm000** [tosdr/edit.tosdr.org#1064] [Ideas] [answered]
Have the ability to sort through different quotes that conflicted either quotes that curators have manually made or Docbot detecting  text witch having multiple methods of having the same case. In addition I imagine there being a mechanism that would automatically decline points that conflict.  For example there may Be a point for no logs approved and then it may say part of the service dose collect logs. This would make the first point invalid. there are other times when this would be more relevant. conflicting point would be automatically declined  if they are for the same case that are approved it would go in a list for  curators  similer to how point that are waiting to be approved go with the option to chose the beater case.

**Comment by JustinBack** [comment, marked as answer]
Added it to the roadmap

https://git.tosdr.org/tosdr/roadmap/issues/2
